packages/table/src/index.ts

```
@@ -0,0 +1,614 @@
+        if (maxWidth <= ellipsisLength) {
+            return string_.slice(0, maxWidth);
+        }
+
+        // Extract ANSI codes and text content
+        const ansiCodes: string[] = [];
+        const ansiPositions: number[] = [];
+        let plainText = string_;
+
+        // Find all ANSI codes and store them with their positions
+        // eslint-disable-next-line no-control-regex,regexp/no-control-character
+        const ansiPattern = /\u001B\[\d+m/g;
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)